docs/examples/1.6.x/client-android/java/functions/get-template.md

```diff
@@ -0,0 +1,22 @@
+import io.appwrite.Client;
+import io.appwrite.coroutines.CoroutineCallback;
+import io.appwrite.services.Functions;
+
+Client client = new Client(context)
+    .setEndpoint("https://cloud.appwrite.io/v1") // Your API Endpoint
```

Comment:
Since we're removing `setEndpoint` from the docs code snippet examples, shouldn't we also remove it here?

Reply:
I see that we are using setEndpoint for the docs examples like [here](https://appwrite.io/docs/references/1.6.x/server-nodejs/users)